reconcile/queries.py

```diff
@@ -787,6 +787,7 @@ def get_jumphosts(hostname: Optional[str] = None) -> JumphostsQueryData:
         soakDays
         mutexes
         sector
+        blockedVersions
```

Comment:
this (and equiv) depends on https://github.com/app-sre/qontract-schemas/pull/479

Reply:
forgot to remove that. snuck in from another PR. thx